fiaas_deploy_daemon/deployer/kubernetes/deployment/deployer.py

```diff
@@ -146,6 +151,10 @@ def _make_volumes(self, app_spec):
 
     def _make_volume_mounts(self, app_spec):
         volume_mounts = []
+        if app_spec.config_maps:
+            for config_map in app_spec.config_maps:
+                volume_mounts.insert(0, VolumeMount(name="{}-config".format(config_map),
```

Comment:
```
$ python -m timeit -s "volumes = []" "volumes.append('a short string')"
5000000 loops, best of 5: 46.1 nsec per loop
$ python -m timeit -s "volumes = []" "volumes.insert(0, 'a short string')"
50000 loops, best of 5: 12.4 usec per loop
```

Inserting at the start is massively more expensive than appending at the end. Since the user has total control over the ordering of these in the spec, I say just append, and document how that translates to precedence when deployed.